includes/controllers/class-admin.php

```diff
@@ -57,6 +57,14 @@ public function __construct( $args = [] ) {
 						'action' => [ $this, 'deactivate_plugin' ],
 						'rest'   => true,
 					],
+
+					'plugin_watermark_action'    => [
+						'base'   => 'admin_base',
+						'path'   => '/watermark-plugin',
+						'method' => 'POST',
```

Comment:
Please change this to a simple GET route, a non-REST one (so you can test this simply by URL).